### DISCUSSION TITLE
PublicoObjetivo

### DISCUSSION THREAD

**Discussion by oscardanielM** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#118] [Q&A] [answered]
Alguien sabe si lo del publico objetivo es algo asi

![image](https://user-images.githubusercontent.com/36013303/155638476-5bcd449d-370d-40c3-bf94-df970a14186a.png)

**Comment by alfonso-ramos** [comment, marked as answer]
La verdad, es que también me encuentro perdido que hacer ahi,  lo que tu hiciste se me hace que es lo mejor y mas convincente de como se tiene que hacer esa practica.

**Comment by doguedogue** [comment]
Exacto, con los valores objetivo

Ej. Ciudad = Mérida

**Reply by roge64** [reply]
Así lo hice yo, claro que con otros datos.  Puedes verlo completo en mi repositorio.

**Reply by oscardanielM** [reply]
vale, muchas gracias.

**Reply by MarioDeLaTorreR** [reply]
Entonces ahí en el mismo recuadro se debe poner así? ej Estado = Veracruz?

**Reply by GloriaCoralCerecedo** [reply]
Yo eso hice, puse varios no solo uno o un rango. Y en los identificadores escogí por grupo no en general. No se si era general o por grupo

**Comment by GloriaCoralCerecedo** [comment]
Así se ve bien, se pueden poner mas opciones como
Ej. Hobies: Leer, Escribir, andar en bici etc.

**Reply by oscardanielM** [reply]
gracias, ayuda mucho.

**Comment by KevinSalgado** [comment]
La verdad yo ando algo confundido, ya que no sé que poner en el área de identificadores ¿alguien sabe?

**Reply by oscardanielM** [reply]
Según lo que entendí, es que tienes que escoger de todos los demás rubros las mejores cualidades para identificar el publico objetivo ideal.

Pero no tienes que escoger muchos porque tu publico general estaría muy extenso y tampoco puedes escoger muy pocas porque el publico en general seria muy pequeño o algo así entendí.

**Reply by KevinSalgado** [reply]
gracias
